newIDE/app/src/InstancesEditor/index.js

```diff
@@ -888,27 +908,16 @@ export default class InstancesEditor extends Component<Props> {
       return;
     }
 
-    if (!hasCursorMovedSinceItIsDown) {
-      // Select the instance that will be moved.
-      this.props.instancesSelection.selectInstance({
```

Comment:
Nice, so this selectInstance was useless here.

Reply:
Yes, actually:
- when Shift is not held, the instance can be selection directly when the mouse button is pressed
- when Shift is held, the pressed instance must only be selected on mouse button release (if not moved)